scripts/write_deployinst_input.py

```diff
@@ -217,63 +217,39 @@ def reactor_render(list, output_file, is_cyborg=False):
     pwr_template = read_template(template_path.replace('[reactor]', 'pwr'))
     mox_reactor_template = read_template(template_path.replace('[reactor]', 'mox'))
     candu_template = read_template(template_path.replace('[reactor]', 'candu'))
+
+    spec_dict = {}
+    # spec_dict[ 'reactor_type'] = [template, assem_size, n_assme_core(for model), n_assem_batch(for model)]
+    spec_dict['BWR'] = [pwr_template, 180, 764 / 1000, 764 / 3000]
+    spec_dict['PHWR'] = [candu_template, int(8000 / 473), 473 / 500, 60]
+    spec_dict['CANDU'] = [candu_template, int(8000 / 473), 473 / 500, 60]
+    spec_dict['PWR'] = [pwr_template, 446, 193 / 1000, 193 / 3000]
+    spec_dict['AP1000'] = [pwr_template, 467, 157, 52]
+    spec_dict['EPR'] = [candu_template, 467, 216, 72]
+
     for data in list:
```

Comment:
you really can't use the word list as a variable. it's special in python because it's a type name.